community/examples/hpc-slurm6-tpu.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: slurm6-tpu
+
+vars:
+  project_id:
+  deployment_name: slurm6-tpu
+  region: us-central1
+  zone: us-central1-b
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  - id: tpu_nodeset
+    source: ./community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu
+    settings:
+      disable_public_ips: false
+      subnetwork_self_link: $(network1.subnetwork_self_link)
+      node_type: v2-8
+      name: v2x8
+      tf_version: 2.10.0
+      preemptible: true
```

Comment:
Please add comment why `preemtible` is enabled.

Reply:
Done.